packages/core/content-type-builder/admin/src/components/FormModal/forms/utils/addItemsToFormSection.js

```diff
@@ -0,0 +1,24 @@
+import has from 'lodash/has';
+/**
+ * @description
+ * Adds form options to the default section or as a new section
+ * @param {array} formTypeOptions Base or advanced options
+ * @param {array} sections The sections to mutate
+ */
+const addItemsToFormSection = (formTypeOptions, sections) => {
+  formTypeOptions.forEach((item) => {
+    if (!has(item, 'sectionTitle')) {
```

Comment:
`if (!item?.sectionTitle)` I don't think lodash is needed here.

Reply:
https://github.com/strapi/strapi/pull/14465/commits/b6bbd3e53465c3a3c4b30a25d5ead9eb78bfc309